helm-chart/csi-driver/values.yaml

```diff
@@ -31,17 +31,17 @@ defaultStorageClass: linode-block-storage-retain
 
 csiProvisioner:
   image: registry.k8s.io/sig-storage/csi-provisioner
-  tag: v3.0.0
+  tag: v5.1.0
   pullPolicy: IfNotPresent
 
 csiAttacher:
   image: registry.k8s.io/sig-storage/csi-attacher
-  tag: v3.3.0
+  tag: v4.7.0
   pullPolicy: IfNotPresent
 
 csiResizer:
   image: registry.k8s.io/sig-storage/csi-resizer
-  tag: v1.3.0
+  tag: v1.12.0
```

Comment:
just double checking, should this be same as that of csi-node-driver-registrar which is v2.12.0 or it is v1.12.0 only?

Reply:
Seems to be correct, latest for csi-resizer is v1.12.0 only and for csi-node-driver-registrar its v2.12.0